app/jobs/course/statistics/assessments_score_summary_download_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class Course::Statistics::AssessmentsScoreSummaryDownloadJob < ApplicationJob
+  include TrackableJob
+  queue_as :lowest
+
+  protected
+
+  def perform_tracked(course, assessment_ids)
+    csv_file = Course::Statistics::AssessmentsScoreSummaryDownloadService.download(course, assessment_ids)
+    redirect_to SendFile.send_file(csv_file,
+                                   "#{Pathname.normalize_filename(course.title)}_assessment_score_summary.csv")
```

Comment:
`course.title` + `score_summary` + `datetimestamp`, no need to add `"assessment_"` since FE doesn't mention "assessment score summary" in the same breath.

Reply:
done